app/api/dependencies.py

```diff
@@ -77,7 +77,7 @@ async def _get_rpc_client(
 async def get_wallet_rpc_client() -> AsyncIterator[WalletRpcClient]:
     async for _ in _get_rpc_client(
         node_type=NodeType.WALLET,
-        self_hostname=settings.CHIA_HOSTNAME,
+        self_hostname=settings.CHIA_WALLET_HOSTNAME or settings.CHIA_HOSTNAME,
```

Comment:
I would write this using ternary
`self_hostname = setting.CHIA_WALLET_HOSTNAME if setting.CHIA_WALLET_HOSTNAME else settings.CHIA_HOSTNAME`

although I'm not sure I have a good reason for it

Reply:
I don't have a good reason for the way I was doing it either, so I changed it to your suggestion